<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopWindow</class>
 <widget class="QMainWindow" name="ShopWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>1044</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShopWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>981</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="InvTab">
     <attribute name="title">
      <string>Inventory</string>
     </attribute>
     <widget class="QPushButton" name="openButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>load inventory</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>439</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Brand</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="brandLWidget">
         <property name="font">
          <font>
           <family>8514oem</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="modelLWidget">
         <property name="font">
          <font>
           <family>8514oem</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>510</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="foundLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Found:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="countLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="sizeminCBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>37</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QComboBox" name="priceminCBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="cartTable">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>610</y>
        <width>451</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>690</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>clear order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkoutButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>770</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>check out</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>590</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addToButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkSki">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ski</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkSnowboard">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Board</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="sizeMax">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>820</y>
        <width>104</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="sizeMin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>820</y>
        <width>104</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEnd">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>400</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateStart">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>400</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="priceMin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>850</y>
        <width>104</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="priceMax">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>850</y>
        <width>104</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sizemaxCBox">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="pricemaxCBox">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>300</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>37</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>max</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>300</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>max</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>151</x>
        <y>51</y>
        <width>28</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>631</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Sort by</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sortCBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="text">
            <string>search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="font">
          <font>
           <family>8514oem</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="tabKeyNavigation">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Brand</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Model</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>580</y>
        <width>217</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Group Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="groupTEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="QueueTab">
     <attribute name="title">
      <string>Reservations and Returns</string>
     </attribute>
     <widget class="QTableWidget" name="orderTWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>251</width>
        <height>571</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Order</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Orders</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>70</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Begin Return</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="returnTWidget">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>150</y>
        <width>256</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>370</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Give</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Returns</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>261</width>
        <height>121</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Order</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Next To Fill:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>137</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
